src/Microsoft.OData.Client/MaterializeFromAtom.cs

```diff
@@ -4,6 +4,8 @@
 // </copyright>
 //---------------------------------------------------------------------
 
+using Microsoft.OData.Edm;
+

```

Comment:
So, Client still has "Atom" related codes, Is there any plan to remove them?

Reply:
This specific class seems to have evolved to handled more than just Atom and I find the name to be a bit misleading due to its usage. It handles all feeds with the right materializer being selected and provided to provide the right representation.